client/my-sites/plugins/eligibility-warnings/index.jsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import SectionHeader from 'components/section-header';
+
+class EligibilityWarnings extends Component {
+	static propTypes = {
+		isEligible: PropTypes.bool.isRequired,
+		errors: PropTypes.arrayOf(
+			PropTypes.shape( {
+				title: PropTypes.string,
+				description: PropTypes.string,
+				actionUrl: PropTypes.string,
+			} )
+		),
+		pluginWarnings: PropTypes.arrayOf(
+			PropTypes.shape( {
+				title: PropTypes.string,
+				description: PropTypes.string,
+				actionUrl: PropTypes.string,
+			} )
+		),
+		widgetWarnings: PropTypes.arrayOf(
+			PropTypes.shape( {
+				title: PropTypes.string,
+				description: PropTypes.string,
+				actionUrl: PropTypes.string,
+			} )
+		),
+		backUrl: PropTypes.string,
+		translate: PropTypes.func,
+	};
+
+	render() {
+		const {
+			translate,
+			backUrl,
+			isEligible,
+			errors,
+			pluginWarnings,
+			widgetWarnings,
+		} = this.props;
+
+		return (
+			<div className="eligibility-warnings">
+				<SectionHeader label={ translate( 'Conflicts' ) } />
+
+				{ errors.map( ( message ) =>
+					<Card key={ message.title } className="eligibility-warnings__message">
+						<Gridicon icon="notice" className="eligibility-warnings__error-icon" />
+						<div className="eligibility-warnings__message-content">
+							<div className="eligibility-warnings__message-title">
+								{ translate( 'Error' ) + `: ${ message.title }` }
```

Comment:
we should probably include this into the translation since it could be handled differently, especially with some languages that are RTL

```js
{ translate( 'Error: %( title )', { args: { title } } }
```